upup/pkg/fi/nodeup/nodetasks/service.go

```diff
@@ -75,8 +74,12 @@ func (p *Service) GetDependencies(tasks map[string]fi.Task) []fi.Task {
 		switch v := v.(type) {
 		case *Package, *UpdatePackages, *UserTask, *GroupTask, *Chattr, *BindMount, *Archive, *Prefix, *UpdateEtcHostsTask:
 			deps = append(deps, v)
-		case *Service, *LoadImageTask, *PullImageTask, *IssueCert, *BootstrapClientTask, *KubeConfig:
+		case *Service, *PullImageTask, *IssueCert, *BootstrapClientTask, *KubeConfig:
```

Comment:
Should this be a separate PR?

Reply:
Yeah, was just playing around to see if it helps. Seems a good idea to keep and put in a separate PR.

Reply:
https://github.com/kubernetes/kops/pull/14613